.github/workflows/build.yml

```diff
@@ -0,0 +1,17 @@
+name: build-all-configs
+on: [push]
+jobs:
+  build-on-osx-for-osx:
+    runs-on: macos-10.15
+    steps:
+    - uses: actions/checkout@v2
+    - id: configure-cmake
+      run: cmake -S . -B build -DDJINNI_WITH_OBJC=ON -DDJINNI_STATIC_LIB=ON -G Xcode
+    - id: build-release
+      run: cmake --build build --parallel $(sysctl -n hw.ncpu) --config Release
+    - id: install-files
+      run: cmake --build build --parallel $(sysctl -n hw.ncpu) --config Release --target install -- DESTDIR=zinstall_root
```

Comment:
What does `zinstall_root` stand for? Is it just a random name or does it have a meaning that I don't get?

Reply:
just some name in which the installation will happen 

it could be install_root, or foobar, but I use the zprefix because ... I do it since ever... maybe to have it listed on the end, I don't know, shall I change it?

Reply:
I don't care, I was just wondering if there was some meaning to it because this naming scheme is new to me.

Reply:
if it confuses you I should possible change it ....  will think about something not so strange with low chance to name collide